Modules/vc-module-export/VirtoCommerce.ExportModule.Core/Model/ExportedTypeDefinition.cs

```diff
@@ -42,6 +42,6 @@ public class ExportedTypeDefinition
         /// Factory function to create a data source for this type
         /// </summary>
         [JsonIgnore]
-        public Func<ExportDataQuery, IPagedDataSource> ExportedDataSourceFactory { get; set; }
+        public IPagedDataSourceFactory DataSourceFactory { get; set; }
```

Comment:
Why these are the changes? I think it is a part of export/dev already. Did you rebase on this branch?

Reply:
Yes. rebased. Agree, it looks strange. May be this is a something due conflict changes.